src/main.cpp

```diff
@@ -1128,3 +1135,41 @@ static bool recompress_map_database(const GameParams &game_params, const Setting
 	actionstream << "Done, " << count << " blocks were recompressed." << std::endl;
 	return true;
 }
+
+static void get_env_opts(Settings &settings)
```

Comment:
I would rename this to `args`, I thought this was writing to the real settings at first look

Reply:
Will do

Reply:
done